stage0/src/dice_attestation.rs

```diff
@@ -0,0 +1,129 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+extern crate alloc;
+
+use alloc::{string::String, vec::Vec};
+use coset::{cbor::value::Value, cwt, iana, CborSerializable, CoseError};
+use hkdf::Hkdf;
+use p384::ecdsa::{signature::Signer, Signature, SigningKey, VerifyingKey};
+use rand_core::OsRng;
+use sha2::Sha256;
+
+
+pub const ID_SALT: &str = "DICE_ID_SALT";
```

Comment:
Please add doc comment for public constants.

Reply:
done